lib/apiservers/engine/backends/container.go

```diff
@@ -165,14 +164,13 @@ func (c *Container) ContainerCreate(config types.ContainerCreateConfig) (types.C
 		config.Config.Entrypoint = layer.Config.Entrypoint
 	}
 	config.Config.Env = append(config.Config.Env, layer.Config.Env...)
-
 	log.Printf("config.Config' = %+v", config.Config)
 
 	// Call the Exec port layer to create the container
-	host, err := guest.UUID()
+	host, err := os.Hostname()
```

Comment:
I noted that I fixed this issue in the wrong place. I will be adding that fix into this PR as well though. another changeset is coming to fix this.